packages/ckeditor5-dev-release-tools/tests/tasks-new/updatedependencies.js

```diff
@@ -0,0 +1,315 @@
+				fs: {
+					readJsonSync: sandbox.stub(),
+					writeJsonSync: sandbox.stub()
+				},
+				glob: {
+					globSync: sandbox.stub()
+				},
+				process: {
+					cwd: sandbox.stub( process, 'cwd' ).returns( '/work/project' )
+				},
+				devUtils: {
+					logger: sandbox.stub().returns( {
+						error: sandbox.stub(),
+						warning: sandbox.stub(),
+						info: sandbox.stub()
+					} )
+				}
+			};
+
+			mockery.enable( {
+				useCleanCache: true,
+				warnOnReplace: false,
+				warnOnUnregistered: false
+			} );
+
+			mockery.registerMock( 'fs-extra', stubs.fs );
+			mockery.registerMock( 'glob', stubs.glob );
+			mockery.registerMock( '@ckeditor/ckeditor5-dev-utils', stubs.devUtils );
+
+			updateDependencies = require( '../../lib/tasks/updatedependencies' );
+		} );
+
+		afterEach( () => {
+			mockery.deregisterAll();
+			mockery.disable();
+			sandbox.restore();
+		} );
+
+		describe( 'preparing options', () => {
+			beforeEach( () => {
+				stubs.glob.globSync.returns( [] );
+			} );
+
+			it( 'should use provided `cwd` to search for packages', () => {
+				const options = {
+					cwd: '/work/another/project'
+				};
+
+				updateDependencies( options );
+
+				expect( stubs.glob.globSync.calledOnce ).to.equal( true );
+				expect( stubs.glob.globSync.getCall( 0 ).args[ 1 ] ).to.have.property( 'cwd', '/work/another/project' );
+			} );
+
+			it( 'should use `process.cwd()` to search for packages if `cwd` option is not provided', () => {
+				updateDependencies( {} );
+
+				expect( stubs.glob.globSync.calledOnce ).to.equal( true );
+				expect( stubs.glob.globSync.getCall( 0 ).args[ 1 ] ).to.have.property( 'cwd', '/work/project' );
+			} );
+
+			it( 'should match only files', () => {
+				updateDependencies( {} );
+
+				expect( stubs.glob.globSync.calledOnce ).to.equal( true );
+				expect( stubs.glob.globSync.getCall( 0 ).args[ 1 ] ).to.have.property( 'nodir', true );
+			} );
+
+			it( 'should always receive absolute paths for matched files', () => {
+				updateDependencies( {} );
+
+				expect( stubs.glob.globSync.calledOnce ).to.equal( true );
+				expect( stubs.glob.globSync.getCall( 0 ).args[ 1 ] ).to.have.property( 'absolute', true );
+			} );
+
+			it( 'should search for packages in `cwd` and `packagesDirectory`', () => {
+				updateDependencies( {
+					packagesDirectory: 'packages'
+				} );
+
+				expect( stubs.glob.globSync.calledOnce ).to.equal( true );
+				expect( stubs.glob.globSync.getCall( 0 ).args[ 0 ] ).to.deep.equal( [
+					'package.json',
+					'packages/*/package.json'
+				] );
+			} );
+
+			it( 'should search for packages only in `cwd` if `packagesDirectory` option is not provided', () => {
+				updateDependencies( {} );
+
+				expect( stubs.glob.globSync.calledOnce ).to.equal( true );
+				expect( stubs.glob.globSync.getCall( 0 ).args[ 0 ] ).to.deep.equal( [
+					'package.json'
+				] );
+			} );
+
+			it( 'should remove leading and trailing path separators from the `packagesDirectory`', () => {
```

Comment:
Looks like tests for removing trailing path separators should be added to a util and not this function.
Su[ch util should be probably used by most of the tasks in the release process that accept `packagesDirectory` as argument.

Reply:
Extracted the `normalizePath()` to `lib/utils/normalizepath.js`, but I would like to keep two tests here to make sure that the path from `packagesDirectory` will be normalized.